### PR TITLE
readme.md and cdn support with rawgit

### PR DESCRIPTION
**Pull request by thinkholic**
Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Login State
+
+Share Login State between the Sub Domains for Meteor Apps
```
Comment (arunoda) [inline README.md]: Add in brackets like this:

(Support static apps too)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## How-To:
```
Comment (arunoda) [inline README.md]: Change this to `Getting Started`.

And also add a description on what user needs to in brief. May be just 2-3 lines.

File: includes/login_state.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+loginState = {};
```
Comment (arunoda) [inline includes/login_state.js]: Make loginState to `LoginState`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+#### Configuration
+
+Update `settings.json` as follows. You need to provide appropiate values for `domain` and `cookineName` fields.
+
+```json
+{
+  "public": {
+    "loginState": {
+      "domain": ".domain.com",
```
Comment (arunoda) [inline README.md]: Add a note on we must start with `.domain.com` to share this will subdomains.

**Comment by thinkholic** [comment]
updated.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+```
+
+### On static app
+
+Include this JavaScript file into your html document.
+
+```javascript
+<script src="https://cdn.rawgit.com/thinkholic/login-state/master/includes/login_state.js" type="text/javascript"></script>
```
Comment (arunoda) [inline README.md]: We may need to change the URL too.

Comment (arunoda) [inline README.md]: @thinkholic use this to submit the file: https://github.com/jsdelivr/jsdelivr

File: README.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+}
+```
+
+#### Installing via bower
+
+`bower install meteor-login-state`
+
+> See: [bower](http://bower.io/)
+
+#### Installing via NPM
```
Comment (arunoda) [inline README.md]: Simply say, "Installing via Browserify"
People know that :)

File: index.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+      while (c.charAt(0)==' ') c = c.substring(1);
+      if (c.indexOf(name) != -1) return c.substring(name.length,c.length);
+  }
+  return "";
+}
+
+if(typeof module !== "undefined") {
+  module.exports = LoginState;
+} else {
+  window.exports = LoginState;
```
Comment (arunoda) [inline index.js]: window.LoginState = LoginState

**Comment by thinkholic** [comment]
Made those changes.

**Comment by arunoda** [comment]
Give a try with browserify.

**Comment by arunoda** [comment]
Publish to both npm and bower

**Comment by thinkholic** [comment]
Published.

On Thu, Jul 2, 2015 at 4:27 PM, Arunoda Susiripala <notifications@github.com
> wrote:

> Publish to both npm and bower
>
>
> On Thu, Jul 2, 2015 at 4:27 PM Arunoda Susiripala <

**Pull request merged**